keiko-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlQueueProperties.kt

```diff
@@ -0,0 +1,49 @@
+package com.netflix.spinnaker.config
+
+import com.netflix.spinnaker.kork.sql.config.SqlRetryProperties
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.validation.annotation.Validated
+import java.time.Duration
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Positive
+
+@ConfigurationProperties("keiko.queue.sql")
+@Validated
+class SqlQueueProperties {
+  /**
+   * Enables use of the SqlQueue implementation, disabling RedisQueue.
+   */
+  var enabled: Boolean = false
+  /**
```

Comment:
nit: does the gradle code formatter not put newlines between these, or is it not turned on for kieko?

Reply:
Must be fine - the spinnaker gradle project is `7.0.1`.